components/dashboard/src/App.tsx

```diff
@@ -371,7 +371,7 @@ function App() {
         toRender = <CreateWorkspace contextUrl={hash} />;
     } else if (isWsStart) {
         toRender = <StartWorkspace workspaceId={hash} />;
-    } else if (/.+?\..+?\/.+?/i.test(window.location.pathname)) {
+    } else if (/^\/[^\/]+?\.[^\/]+?\/.+?/i.test(window.location.pathname)) {
```

Comment:
@iQQBot, I think it would be less error-prone to test if the `pathname` starts with either `github.com/` or `gitlab.com/`. that's also what we used to have in nginx ages ago and are now trying to replicate in dashboard code.